jobserver/urls.py

```diff
@@ -52,6 +53,11 @@
         WorkspaceStatusesAPI.as_view(),
         name="workspace-statuses",
     ),
+    path(
+        "api/v2/workspaces/<workspace_name>/releases/",
+        ReleaseUploadAPI.as_view(),
+        name="upload-release",
```

Comment:
Can you prefix the name with `workspace`, keeps it in line with all the other URL names.